source/includes/_endpoint-dataset.md

```diff
@@ -149,15 +149,24 @@ options.  The limit and offset are returned in relationship to datasets related
 
 Here are the parameters that can be passed to the search endpoint.
 
-Parameter                  | Type        | Description
----------------------------|-------------|------------------------------------------------
-q                          | string      | query string
-f                          | json Object | used to filter the output of the search (see below)
-limit                      | integer     | limit the number of dataset results returned by the api to less than this amount (default: 10)
-offset                     | integer     | offset into the search index to start gathering results from pre-filter
-max_variables_per_dataset  | integer     | limit the number of variables that match to this number (default: 1000, max: 1000)
-projection                 | json Array  | used to limit the fields that should be returned in the search results. ID is always provided.
-scope                      | json Array  | used to limit the fields that the search should look at.
+Parameter                          | Type        | Description
+-----------------------------------|-------------|------------------------------------------------
+q                                  | String      | query string
+f                                  | Json Object | used to filter the output of the search (see below)
+limit                              | Integer     | limit the number of dataset results returned by the api to less than this amount (default: 10)
+offset                             | Integer     | offset into the search index to start gathering results
+max_variables_per_dataset          | Integer     | limit the number of variables that match to this number (default: 1000, max: 1000)
+embedded_variables                 | Boolean     | embed the results within the dataset results (this will become the default in the future)
+max_subfield_entries_per_variable  | Integer     | some fields in a variable result are a list of items, these lists can be very long at times (think category names).  This limits the number of results for performance/noise reduction.  Pertinent (matching) results are returned first, and then remaining results are padded to meet the limit.  
+projection                         | Json Object | used to limit the fields that should be returned in the search results. ID is always provided.
+scope                              | Json Object | used to limit the fields that the search should look at.
+
+<aside class="notice">
+By default there are only 10 datasets returned.  Inside the response you will find a `totals.datasets` that
```

Comment:
Should probably also point out user should never paginate on `totals.variables` when `embedded_variables=False` as it is instead computed by counting the matches per dataset so it's not a reliable source for "total existing variables", but only for "total variables for the 10 returned datasets"